src/convertToObject.js

```diff
@@ -5,8 +5,21 @@
  *
  * @return {object}
  */
+
 function convertToObject(sourceString) {
-  // write your code here
+  const obj = sourceString
+    .split(';')
+    .filter((line) => line.trim() !== '')
+    .reduce((object, el) => {
+      let [key, value] = el.split(':').map((item) => item.trim());
+
+      key = key.split('').join('').trim();
+      value = value.split('').join('').trim();
+
+      return Object.assign(object, { [key]: value });
```

Comment:
[CODE STYLE] - All chained methods called on the same level should have the same indentation. This makes code much more readable. Also, ensure that the `reduce` method's callback function is properly indented to maintain readability.